src/features/projects/components/ProjectSnippet.tsx

```diff
@@ -94,24 +99,61 @@ export default function ProjectSnippet({
           ></div>
         ) : null}
         {project.isTopProject && project.isApproved && (
-          <div className={'topProjectBadge'}>{t('common:topProject')}</div>
+          <>
```

Comment:
1. Please separate the top project badge into a different component.
2. The figma design seems to have changed slightly. @mariahosfeld could you confirm this? Should those changes be made here as well?
3. I think the border on the top project badge can be removed. It isn't in the current figma design as well.
<img width="357" alt="Screenshot 2023-05-24 at 10 51 11 AM (2)" src="https://github.com/Plant-for-the-Planet-org/planet-webapp/assets/44917347/1d0963f1-3ac9-4bd3-87f3-6295bdabaac0">